docs/source/user-guide/tutorials/build-r-pkgs.rst

```diff
@@ -0,0 +1,295 @@
+package hosted on CRAN that is used for examining and
+cleaning up data.
+
+To begin building it, type:
+
+.. code-block:: bash
+
+    conda skeleton cran janitor
+
+This creates a directory named ``r-janitor`` and
+creates one skeleton file in that directory: ``meta.yaml``.
+Many other files can be added there as necessary, such
+as ``build.sh`` and ``bld.bat``, test scripts, or anything else
+you need to build your software. Use the ``ls`` command
+to verify that this file has been created. The meta.yaml
+file has been populated with information from the CRAN
+metadata and in many cases will not need to be edited.
+
+Now that you have the conda build recipe ready, you can
+use conda build to create the package:
+
+.. code-block:: bash
+
+    conda-build r-janitor
+
+What may happen at this point is that you will have
+dependencies of this package that do not exist as conda
+packages yet. They need to be turned into conda packages.
+Use conda skeleton to recursively build out recipes for
+the packages that it depends on:
+
+.. code-block:: bash
+
+    conda skeleton cran janitor --recursive
+
+You can manually build each package individually
+by typing:
+
+.. code-block:: bash
+
+    conda-build package-name
+
+.. note::
+   Replace "package-name" with the name of each
+   package.
+
+Once all of the package dependencies are resolved, you
+can build the R package by using:
+
+.. code-block:: bash
+
+    conda-build .
+
+Now you can install your newly built package in your
+conda environment by using the use-local flag:
+
+.. code-block:: bash
+
+    conda install --use-local r-janitor
+
+
+The remaining optional sections show you how to make
+R packages for other R versions and other architectures
+and how to upload them to your Anaconda.org account.
+
+.. _`different-r-version`:
+
+Optional---Building for a different R version
+=============================================
+
+By default, conda build creates packages for the version
+of R installed in the root environment. To build packages
+for other versions of R, you use the `--r` flag followed by
+a version.
+
+For example, to explicitly build a version of the fansi package
+for R 3.5.3, use:
+
+.. code-block:: bash
+
+    conda-build --r 3.5.3 r-fansi
+
+Notice that the file printed at the end of the conda-build
+output has changed to reflect the requested version of R.
+Conda install will look in the package directory for the file
+that matches your current R version.
+
+Example file path:
+
+.. code-block:: text
+
+    /Users/jsmith/anaconda3/conda-bld/osx-64/r-fansi-0.4.0-r351h46e59ec_0.tar.bz2
+ 
+.. note::
+   Your path and filename will vary depending on your
+   installation and operating system. Save the path and
+   filename information for the next task.
+
+.. _`upload-to-anaconda-org1`:
+
+Optional---Uploading packages to Anaconda.org
+=============================================
+Anaconda.org, formerly known as binstar.org, is a
```

Comment:
binstar.org has been gone for so long that it's probably not worth mentioning here in new content.